src/client/pps.go

```diff
@@ -588,17 +588,13 @@ func (c APIClient) StopPipeline(name string) error {
 	return grpcutil.ScrubGRPC(err)
 }
 
-// RerunPipeline reruns a pipeline over a given set of commits. Exclude and
-// include are filters that either include or exclude the ancestors of the
-// given commits.  A commit is considered the ancestor of itself. The behavior
-// is the same as that of ListCommit.
-func (c APIClient) RerunPipeline(name string, include []*pfs.Commit, exclude []*pfs.Commit) error {
-	_, err := c.PpsAPIClient.RerunPipeline(
+// RunPipeline runs a pipeline. It can be passed a list of commit provenance. This will
```

Comment:
This comment got truncated or something.

Reply:
I think I just got distracted while typing this lol